clang/include/clang/Driver/Options.td

```diff
@@ -2876,6 +2876,15 @@ def flax_vector_conversions : Flag<["-"], "flax-vector-conversions">, Group<f_Gr
 def flimited_precision_EQ : Joined<["-"], "flimited-precision=">, Group<f_Group>;
 def fapple_link_rtlib : Flag<["-"], "fapple-link-rtlib">, Group<f_Group>,
   HelpText<"Force linking the clang builtins runtime library">;
+
+/// ClangIR-specific options - BEGIN
+def fclangir_enable : Flag<["-"], "fclangir-enable">, Visibility<[ClangOption, CC1Option]>,
```

Comment:
Most of our options aren't spelled with `enable` -- instead we use `-fname` and `-fno-name` as a pair; so perhaps `-fclangir` and `-fno-clangir` would make more sense?

Reply:
Even better :)